src/analysis/functions.rs

```diff
@@ -349,6 +521,10 @@ fn analyze_function(
     }
 
     if !commented {
+        if !destroys.is_empty() || !callbacks.is_empty() {
+            imports.add("std::boxed::Box as Box_", None);
```

Comment:
`Box` should only be needed for callbacks that are not `scope="call"`

Reply:
This is a bit more tricky than that. I think I'll do it as a second time (outside of this PR) as a more global imports cleanup.

Reply:
Added a `TODO`.